.github/workflows/integrate.yml

```diff
@@ -0,0 +1,158 @@
+# master only
+
+name: Integrate
+
+on:
+  push:
+    branches: [master]
+
+env:
+  SLS_IGNORE_WARNING: '*'
+  FORCE_COLOR: 1
+
+jobs:
+  linuxNode14:
+    name: '[Linux] Node.js 14: Unit tests '
+    runs-on: ubuntu-latest
+    steps:
+      - name: Resolve last validated commit hash (for `git diff` purposes)
+        env:
+          # See https://github.com/serverlessinc/setup-cicd-resources
+          GET_LAST_VALIDATED_COMMIT_HASH_URL: ${{ secrets.GET_LAST_VALIDATED_COMMIT_HASH_URL }}
+          PUT_LAST_VALIDATED_COMMIT_HASH_URL: ${{ secrets.PUT_LAST_VALIDATED_COMMIT_HASH_URL }}
+        run: |
+          curl -f "$GET_LAST_VALIDATED_COMMIT_HASH_URL" -o /home/runner/last-validated-commit-hash || :
+          curl -X PUT -H "User-Agent:" -H "Accept:" -H "Content-Type:" -d "$GITHUB_SHA" "$PUT_LAST_VALIDATED_COMMIT_HASH_URL"
+      - name: Store last validated commit hash (as it's to be used in other job)
+        uses: actions/upload-artifact@v2
+        with:
+          name: last-validated-commit-hash
+          path: /home/runner/last-validated-commit-hash
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Node.js and npm
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+          registry-url: https://registry.npmjs.org
+
+      - name: Retrieve dependencies from cache
+        id: cacheNpm
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.npm
+            node_modules
+          key: npm-v14-${{ runner.os }}-${{ github.ref }}-${{ hashFiles('package.json') }}
+          restore-keys: npm-v14-${{ runner.os }}-${{ github.ref }}-
+
+      - name: Install dependencies
+        if: steps.cacheNpm.outputs.cache-hit != 'true'
+        run: |
+          npm update --no-save
+          npm update --save-dev --no-save
+      - name: Unit tests
+        # Some tests depend on TTY support, which is missing in GA runner
+        # Workaround taken from https://github.com/actions/runner/issues/241#issuecomment-577360161
+        run: script -e -c "npm test -- -b"
```

Comment:
Nit: This workaround was important in `serverless` but I think no other repo needed that.

Reply:
You're right - removed